_pages/en_US/include/super-skaterhax.txt

```diff
@@ -0,0 +1,66 @@
+
+The instructions provided only apply to the New 3DS, New 3DS XL, and New 2DS XL. Please ensure that the device you are modding is a New 3DS, New 3DS XL, or New 2DS XL before continuing.
+{: .notice--warning}
+
+### What You Need
+
+* The latest release of [Luma3DS](https://github.com/LumaTeam/Luma3DS/releases/latest) (the Luma3DS `.zip` file)
+* The latest release of [super-skaterhax](https://github.com/zoogie/super-skaterhax/releases) (the release `.zip` file)
+
+#### Section I - Updating the System
+
+{% include_relative include/sysupdate.txt cfw="false" %}
+
+#### Section II - Prep Work
+
+In this section, you will copy the files needed to trigger both browserhax and the Homebrew Launcher.
+
+1. Power off your device
+1. Insert your SD card into your computer
+1. Copy `boot.firm` and `boot.3dsx` from the Luma3DS `.zip` to the root of your SD card
+  + The root of the SD card refers to the initial directory on your SD card where you can see the Nintendo 3DS folder, but are not inside of it
+1. Copy all files inside the folder for your device's region in the super-skaterhax `.zip` to the root of your SD card
+1. Reinsert your SD card into your device
+1. Power on your device
+
+#### Section III - super-skaterhax
+
+In this section, you will visit the browser exploit webpage, which will launch the Homebrew Launcher.
+
+If you haven't already, make sure you have a working Internet connection set up on your device.
+{: .notice--info}
+
+1. On the HOME Menu, press the Left and Right shoulder buttons together to open the camera
+    + If you are unable to open the camera, open the Internet Browser and manually type the URL instead (`https://zoogie.github.io/web/nbhax/`)
+1. Tap the QR code button and scan one of the following QR codes for your device's region [here](https://user-images.githubusercontent.com/28328903/226086338-585bfdac-0aac-44c0-b413-89206d2815d8.png)
+  + If you get an error, [follow this troubleshooting guide](troubleshooting#installing-boot9strap-ssloth-browser)
+  + If you get a security certificate warning, press (A) to allow the connection
+1. Press the (Select) button to open the bookmark tab
+  + If your (Select) button does not respond, tap the star icon on the bottom-left corner of the screen
+1. Tap "Bookmark this page"
+1. Press (B) once to return to the browser
+1. Press (Start) to open the context menu
+  + If your (Start) button does not respond, tap the 3-line menu on the bottom-right corner of the screen
+1. Tap on "Settings"
+1. Tap on "Delete Cookies"
+1. Press (A) to continue
+1. Press (Home) to return to the HOME Menu, then immediately press (A) to launch the browser again
+1. Select the "GO GO!" button on the top screen
+  + If any prompts appear, approve all of them
+1. If the exploit was successful, your device will have booted into the Homebrew Launcher
+    + For further exploits on this guide, you need to launch the Homebrew Launcher again - to do this, simply redo this section
```

Comment:
i don't think this line is necessary

Reply:
I would think it causes confusion but okay I guess

Reply:
if its not there*